dotcom-rendering/src/components/InlineProductCard.stories.tsx

```diff
@@ -0,0 +1,123 @@
+import { css } from '@emotion/react';
+import { breakpoints } from '@guardian/source/foundations';
+import type { Meta } from '@storybook/react-webpack5';
+import { ArticleDesign, ArticleDisplay, Pillar } from '../lib/articleFormat';
+import type { ProductImage } from '../types/content';
+import { ArticleContainer } from './ArticleContainer';
+import type { InlineProductCardProps } from './InlineProductCard';
+import { InlineProductCard } from './InlineProductCard';
+import { Section as SectionComponent } from './Section';
+
+const meta = {
+	component: InlineProductCard,
+	title: 'Components/InlineProductCard',
+	parameters: {
+		chromatic: {
+			viewports: [
+				breakpoints.mobile,
+				breakpoints.tablet,
+				breakpoints.wide,
+			],
+		},
+
+		formats: [
+			{
+				design: ArticleDesign.Standard,
+				display: ArticleDisplay.Standard,
+				theme: Pillar.Lifestyle,
+			},
+		],
```

Comment:
Do you need to set `formats` given it's already set in the `format` arg?

```suggestion
```

See the docs on colour scheme here:

https://github.com/guardian/dotcom-rendering/blob/0e3c2ba03182eddccfae45295bbe15c76748ae1c/dotcom-rendering/.storybook/toolbar/globalColourScheme.ts#L69-L78

This documentation could be better, I've written this for us to take a look at improving it: #14781.